### PR TITLE
adding support for environment variables as markup

### LINKED ISSUE #149: Use environment variables in swagger.yml
Is it possible? If not is there a work around?

Specifically, I would like to do something like this:

```yaml
---
  swagger: "2.0"
  info:
    title: "API"
    description: "AR dev REST API"
    version: "1.0.0"
  host: ${API_HOST}
```

### PR DESCRIPTION
**Pull request by mtrovo**
Fixes #149

### CODE REVIEW

**Comment by kailuowang** [comment]
Thanks so much!  This looks great to me. 
Shall we add the facilities to allow users to config a serials of `OutputTransformers` in a new sbt settingKey? Also maybe add to the FAQ about this capability with an example of enabling Envrionment variables markup?

**Comment by mtrovo** [comment]
Hey @kailuowang thanks for the feedback, I think it's a good idea otherwise
there's no way to use it to generate static files, do you have any idea
about the name for this setting key?

I'm planning to address both comments on a commit later today.

**Comment by kailuowang** [comment]
I am bad at naming too. How about `swaggerOutputTransformers` : `List[String]` ?

File: README.md [hunk 1]
```diff
@@ -100,6 +100,13 @@ Also in build.sbt add domain package names for play-swagger to auto generate swa
 swaggerDomainNameSpaces := Seq("models")
 ```
 
+You can also use markup on your swagger files by providing OutputTransformers classes to the variable `swaggerOutputTransformers` on your build file.
```
Comment (kailuowang) [inline README.md]: "variable" -> "setting" ?

File: README.md [hunk 2]
```diff
@@ -100,6 +100,13 @@ Also in build.sbt add domain package names for play-swagger to auto generate swa
...
+
+For example you can use environment variables by adding the configuration:
+```
+    swaggerOutputTransformers := Seq("com.iheart.playSwagger.EnvironmentVariablesTransformer")
+```
+
```
Comment (kailuowang) [inline README.md]: How about adding an example of the routes file using this as well? Like 
```
###
#  summary: Get the track metadata
#  responses:
#    200:
#      summary: ${PATH}
#      schema:
#        $ref: '#/definitions/namespace2.Track'
###
```

**Comment by kailuowang** [comment]
Thanks very much!

**Pull request merged**